### ISSUE TITLE
Aggressive NPC Pathfinding

### ISSUE THREAD

**Issue by Lerl** [lare96/asteria-3.0#33]
It appears as if when you declare an npc to be aggressive, and you go near it, it just stands there and can't find its way to you, even when nothing is in the way.

**Comment by lare96** [comment]
A little confused, there aren't any pathfinding algorithms present?

**Comment by Lerl** [comment]
Maybe this will help:

https://www.youtube.com/watch?v=qKNq8UrinG4&feature=youtu.be

**Comment by lare96** [comment]
Yeah that's not pathfinding, but I'll definitely check it out. What triggers it?

**Comment by Lerl** [comment]
Nothing appears to trigger it, it is what happens when you spawn an npc and try to get it to find you.
